src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -642,11 +642,11 @@ public InventoryMock createInventory(InventoryHolder owner, @NotNull InventoryTy
 		case BREWING:
 			return new BrewerInventoryMock(owner);
 		case CRAFTING:
-			// TODO: This Inventory Type needs to be implemented
+			throw new UnsupportedOperationException("Crafting Inventory can't be created");
 		case CREATIVE:
-			// TODO: This Inventory Type needs to be implemented
+			throw new UnsupportedOperationException("Creative Inventory can't be created");
 		case MERCHANT:
-			// TODO: This Inventory Type needs to be implemented
+			throw new UnsupportedOperationException("Merchant Inventory can't be created");
```

Comment:
There is already an if statement guarding the switch so these cases can never be reached
```suggestion
```